saos-persistence/src/main/java/pl/edu/icm/saos/persistence/model/RemovedJudgment.java

```diff
@@ -0,0 +1,63 @@
+package pl.edu.icm.saos.persistence.model;
+
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+
+import org.joda.time.DateTime;
+
+import pl.edu.icm.saos.persistence.common.DataObject;
+
+@Entity
+@SequenceGenerator(name = "seq_removed_judgment", allocationSize = 1, sequenceName = "seq_removed_judgment")
+public class RemovedJudgment extends DataObject {
+
+    private long removedJudgmentId;
+    
+    private JudgmentSourceInfo sourceInfo = new JudgmentSourceInfo();
+    
+    private DateTime removedDate = new DateTime();
```

Comment:
I'm not sure if the date is needed. DataObject has 'creationDate' which is going to be the same - the date of creation of the record with deleted judgment is the date when it was deleted.